### PR TITLE
feat: design contact page

### PR DESCRIPTION
**Pull request by melvin78**
### Description

_Add a short description of the changes._

### Merge Checklist

- [x] Is the title of the PR in this format: `[fix|docs|feat|refactor|chore|tests] short description` ?
- [x] Added new tests - if necessary (if you changed an interface, you most probably need to add new tests)
- [x] Tests are passing?
- [x] Added appropriate documentation (if you changed an interface or added a new resource, you most probably need to update/add the appropriate documentation)
- [x] Do you need to update the README?

### CODE REVIEW

File: components/Contact.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            </button>
+          </div>
+          <div class="p-2 w-full pt-8 mt-8 border-t border-gray-200 text-center">
+            <p class="leading-normal my-5">
+              Nairobi
+              <br>Kenya,
+            </p>
+            <span class="inline-flex">
+              <a class="text-gray-500">
+                <svg fill="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="w-5 h-5" viewBox="0 0 24 24">
```
Comment (dxphilo) [inline components/Contact.vue:47]: One this, can you please separate these svg into separate component inside the `components/icons`  directory and import them here ?. Overal LGTM.

File: components/Contact.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                  <path d="M18 2h-3a5 5 0 00-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 011-1h3z" />
+                </svg>
+              </a>
+              <a href="https://twitter.com/KenyaVue" target="_blank" class="ml-4 text-gray-500">
+                <svg fill="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="w-5 h-5" viewBox="0 0 24 24">
+                  <path d="M23 3a10.9 10.9 0 01-3.14 1.53 4.48 4.48 0 00-7.86 3v1A10.66 10.66 0 013 4s-4 9 5 13a11.64 11.64 0 01-7 2c9 5 20 0 20-11.5a4.5 4.5 0 00-.08-.83A7.72 7.72 0 0023 3z" />
```
Comment (dxphilo) [inline components/Contact.vue:53]: Same here

Comment (melvin78) [reply components/Contact.vue:53]: sure no problem

Comment (melvin78) [reply components/Contact.vue]: done

**Review by dxphilo — APPROVED** [review]

**Pull request merged**